### PR TITLE
Added documentation 

### PR DESCRIPTION
**Pull request by andreaarmanni**
Added documentation only related to market deployment whilst kept documentation about components on main ocean docs

### CODE REVIEW

**Comment by andreaarmanni** [comment]
@mihaisc removed relative URLs and updated links

**Comment by andreaarmanni** [comment]
> Can you also update the links on forking the market (in [this section](https://github.com/oceanprotocol/market?tab=readme-ov-file#-forking)) to link to these docs. The links currently go to gitbook, if the documentation is going to be in the repo then we should also update the links.

Thanks for the feedback Jamie, all links pointing to Gitbook are removed now

**Comment by jamiehewitt15** [comment]
> all links pointing to Gitbook are removed now

Can you double-check that? There are no edits to the main readme in this PR. I'm talking about the links (in [this section](https://github.com/oceanprotocol/market?tab=readme-ov-file#-forking))

File: documentation/build-a-marketplace/README.md [hunk 1]
```diff
@@ -19,9 +19,9 @@ Using Ocean Market is already a big improvement on the alternatives that are out
 
 The tutorial covers:
 
-* [Forking and running Ocean Market locally](forking-ocean-market.md)
-* [Customizing your fork of Ocean market](customising-your-market.md)
-* [Quick deployment of Ocean Market](deploying-market.md)
```
Comment (jamiehewitt15) [inline documentation/build-a-marketplace/README.md]: These links to the other files are useful, I would suggest keeping them. Just double check they work

**Pull request merged**